README.md

```diff
@@ -20,6 +20,25 @@ ReactJS Dallas is a safe place for anyone who wants to help build and develop an
 
 The default Gatsby starter.
 
+## Development Setup
+
+1. Fork and Clone Repository: `git clone --depth=1 https://github.com/<your-username>/reactjs-dallas-site.git`
```

Comment:
should we de-dupe this with the contributing.md instructions?

Reply:
Yessir, I'll merge this and update the docs in a subsequent pull request.